gulp/tasks/webpack.js

```diff
@@ -0,0 +1,68 @@
+'use strict';
+
+var gulp = require('gulp'),
+    gulpUtil = require('gulp-util'),
+    webpack = require('webpack'),
+    WebpackDevServer = require('webpack-dev-server'),
+    gitUtils = require('../util/gitUtils'),
+    config = require('../config.js'),
+    wpConfig = require('../../webpack.config.js'),
+    wpConfigFactory = wpConfig.configFactory,
+    wpConfigDevServer = wpConfig.devServerConfig;
+
+/* Builds the pattern library doc into /pldoc/public.
```

Comment:
Nit: I've never seen multi-line comments like this in our codebase. We either use ``//`` or do a JSDoc-style comment starting with ``/**``:

http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/javascript_guidelines.html#documentation

Not a big deal if you prefer doing it like this, but just fyi.